### PR TITLE
CATROID-907 "Report __" brick

### PR DESCRIPTION
**Pull request by Rawbat**
Implemented brick with similar functionality like stop script brick for now. Also included a CLT

### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroid/blob/develop/README.md) and [wiki pages](https://github.com/Catrobat/Catroid/wiki/) of this repository.

- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed
- [x] Post a message in the *catroid-stage* or *catroid-ide* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

**Review by hofcle78 — APPROVED** [review]

File: catroid/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -881,6 +881,8 @@
     <string name="brick_user_defined_script_as" tools:ignore="UnusedResources">as</string>
 
     <string name="brick_user_defined_list_empty">Tap \"+\" to add your own new bricks.</string>
+
+    <string name="brick_report">Report</string>
```
Comment (wslany) [inline catroid/src/main/res/values/strings.xml:885]:
```suggestion
    <string name="brick_report" tools:ignore="UnusedResources">Report</string>
```
For CatBlocks

**Review by wslany — CHANGES_REQUESTED** [review]
Please increase the Catrobat language version in constants.

Please eliminate the "Show variable" brick in the CLT:
image

File: catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java [hunk 2]
```diff
@@ -693,7 +691,8 @@ public void userBricks() {
 		assertEquals(arLocale.getDisplayLanguage(), Locale.getDefault().getDisplayLanguage());
 		assertTrue(RtlUiTestUtils.checkTextDirectionIsRtl(Locale.getDefault().getDisplayName()));
 		openCategory(R.string.category_user_bricks);
-		onView(allOf(withId(android.R.id.list), withParent(withId(R.id.fragment_user_defined_brick_list)))).check(matches(hasChildCount(0)));
+
+		checkIfBrickISRtl(ReportBrick.class, R.id.brick_report_layout);
```
Comment (wslany) [inline catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java:695]: Tests fail with
```
06:17:13  /home/catroid/jenkins_slave/workspace/Catroid_PR-3990/catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java:695: error: cannot find symbol
06:17:13  		checkIfBrickISRtl(ReportBrick.class, R.id.brick_report_layout);
06:17:13  		                  ^
06:17:13    symbol:   class ReportBrick
06:17:13    location: class RtlBrickTest
```
see https://jenkins.catrob.at/job/Catroid/job/PR-3990/16/execution/node/79/log/

Probably due to https://github.com/Catrobat/Catroid/blob/5f6f631fa48bca33e78f02a8152585c52a9d6437/catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java#L248
i.e., you need the resource also in the arabic strings.xml resources.

Comment (Rawbat) [reply catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/rtl/RtlBrickTest.java:696]: Should be working now

**Review by wslany — APPROVED** [review]

**Comment by wslany** [comment]
Thanks!

**Pull request merged**